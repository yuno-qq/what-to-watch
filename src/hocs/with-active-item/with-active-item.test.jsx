import React from 'react'
import {describe, test, expect} from "@jest/globals"
import renderer from 'react-test-renderer'

import withActiveItem from "./with-active-item.jsx"


describe(`<withActiveItem> snapshots:`, () => {
  test(`should render component`, () => {
    const MockComponent = withActiveItem(() => <div className="MOCK_COMPONENT"></div>)

    const tree = renderer.create(<MockComponent />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
