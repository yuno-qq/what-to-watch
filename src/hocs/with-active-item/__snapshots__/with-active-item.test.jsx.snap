// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<withActiveItem> snapshots: should render component 1`] = `
<div
  className="MOCK_COMPONENT"
/>
`;
